CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Fixes
 - Truncate `content` and `header` of `ListItem` when used from `DropdownSelectedItem` @silviuavram ([#1161](https://github.com/stardust-ui/react/pull/1161))
+- `FocusTrapZone` - Do not propagate any keyboard events @sophieH29 ([#1180](https://github.com/stardust-ui/react/pull/1180))
```

Comment:
could you, please, provide some notes about resolution strategy in this PR's description - I mean, specifically
- what was the cause of the issue
- why this was considered as a fix

This description will help a lot to better understand why provided changes are considered to be a fix for the issue, as well as to retrospect this case if we will experience similar problem in future. Thank you!